lib/ood_core/job/adapters/slurm.rb

```diff
@@ -178,6 +179,18 @@ def get_jobs(id: "", owner: nil, attrs: nil)
             return [{ id: id, state: 'undetermined' }]
           end
 
+          def accounts
+            user = Etc.getlogin
+            args = ['-np', 'show', 'accounts', 'withassoc', 'format=account,user', '-P']
```

Comment:
```suggestion
            args = ['-nP', 'show', 'users', 'withassoc', 'format=account,user', 'where', "user=#{user}"]
```

Reply:
Thanks, I'm going to commit the suggestion and then fix the tests it likely breaks.